src/components/atoms/UserStatusDropdown/UserStatusDropdown.tsx

```diff
@@ -5,19 +5,34 @@ import { USER_STATUSES } from "settings";
 
 import { useProfileStatus } from "hooks/useProfileStatus";
 
+import { UserStatus } from "types/User";
+
 import "./UserStatusDropdown.scss";
 
-export const UserStatusDropdown: React.FC = () => {
+export interface UserStatusDropdownProps {
+  userStatuses: UserStatus[];
```

Comment:
But that'd indicate that this component can be used without the prop, right? Which would be slightly miss-loading since without the prop nothing will show up, because of the empty array default value.

Reply:
@0xdevalias @mike-lvov Can one of you guys give some input here please?

Reply:
Given there’s no other props and it’s presumably always intended to be passed in, probably fine to leave as is.

No strong opinion on prop name as I don’t know enough about the feature/where it comes from to make an informed choice.